sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -828,8 +828,10 @@ export interface Location {
     unavailable?: boolean;
 }
 
+export { Location_2 as Location }
```

Comment:
did we file a bug report with them? who can we tag here to take a look?

Reply:
I don't know whether we dug further.  @KarishmaGhiya you might remember?

Reply:
I think @mikeharder ended up pinning us to this old version because of this change.

The problem is that certain interface names are part of standard TS libraries. `Location` is declared by `dom` if I remember correctly. 

Since the name conflicts with a global lib interface, it declares it with a different name to avoid this declaration merging/extending the global one.

Reply:
See https://github.com/Azure/azure-sdk-for-js/issues/9410

Reply:
I would recommend closing this PR based on our prior findings documented in #9410
The noise this would bring to our API reviews is not desirable
I'd like to understand the reason behind the change in behavior, what our options here are etc